### ISSUE TITLE
TBAlwaysSetPreference-allowChangesToManuallySetNetworkSettings does not work

### ISSUE THREAD

**Issue by chipitsine** [Tunnelblick/Tunnelblick#413]
according to https://groups.google.com/forum/#!topic/tunnelblick-discuss/b5UdQtcU8RY 

I made the following Info.plist

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>TBPackageVersion</key><string>1</string>
    <key>TBAlwaysSetPreference-notOKToCheckThatIPAddressDidNotChangeAfterConnection</key><true/>
    <key>TBAlwaysSetPreference-allowChangesToManuallySetNetworkSettings</key><true/>
</dict>
</plist>
```

however, on 3.7.5beta01 it says that setting is unknown

can you provide the proper Info.plist example ?
also, is there a safe way of providing the setting to earlier clients (I do not want to get warning on earlier tunnelblick) ?

**Comment by jkbullard** [comment]
It's a bug: that preference wasn't added to the whitelist. I have a commit to fix it but won't be merging if for a couple days or so; it will be included in the next beta.

I don't think there is a way to provide it to older versions of Tunnelblick -- that would defeat the purpose of a whitelist! I'm not sure there's a good way to solve the problem; the reason for using a whitelist is that it is the safest way validate the preferences.

Allowing older versions of Tunnelblick to "accept" preferences that they don't understand seems like a bad idea in general, but it might be OK to make the error into a warning and allow the installation of the configuration to proceed. I'll think about that.

In the case of the `-allowChangesToManuallySetNetworkSettings` preference in older versions of Tunnelblick, if the settings weren't set manually the preference isn't needed, and if they were set manually the preference won't work! (At least in _some_ older clients: those before it was implemented. It would work in the versions of Tunnelblick that include the implementation but don't have the preference on the whitelist.)

**Comment by chipitsine** [comment]
as for old versions of Tunnelblick, I think the acceptable behaviour would be to ignore what is not supported.

there's no way to upgrade all of users at a time, so I do not want to bother older versions.

**Comment by jkbullard** [comment]
> It's a bug: that preference wasn't added to the whitelist. I have a commit to fix it but won't be merging if for a couple days or so; it will be included in the next beta.

This has been fixed in Tunnelblick 3.7.5beta04.
